core/frontend/src/render/GraphicBranch.ts

```diff
@@ -160,6 +160,10 @@ export interface GraphicBranchOptions {
    * @internal
    */
   viewAttachmentId?: Id64String;
+  /** If true, the view's [[DisplayStyleSettings.clipStyle]] will be disabled for this branch.
+   * No [[ClipStyle.insideColor]], [[ClipStyle.outsideColor]], or [[ClipStyle.intersectionStyle]] will be applied.
+   */
+  disableClipStyle?: boolean;
```

Comment:
Consider changing the type to `true | undefined`. Otherwise, you will have to handle the case where you push a branch that disables the clip style, containing another branch that re-enables it, which doesn't seem useful.

Reply:
done in latest commit